pkgs/tools/misc/bonk/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "bonk";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "elliot40404";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-Y6Hia+B7kIvdvpuZwWGJBsn+pOBmMynXai4KWkNs4ck=";
```

Comment:
`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21